zeroex/orderwatch/decoder/event_decoder.go

```diff
@@ -609,6 +609,21 @@ func (e UntrackedTokenError) Error() string {
 	return fmt.Sprintf("event for an untracked token: contract address: %s, topic: %s", e.TokenAddress.Hex(), e.Topic.Hex())
 }
 
+// AbiParserError is thrown when the decoder fails to parse topics from a
+// retrieved log.
+// NOTE(oskar): this error occurs during abi.ParseTopics and can be a result of
+// certain tokens not conforming to the ERC20 event standard, for example not
+// using indexed log parameters.
+type AbiTopicParserError struct {
```

Comment:
fwiw, i think `AbiParseTopicError` would be a better name, since `TopicParser` isn't the name of a struct.